conf/messages

```diff
@@ -314,12 +314,12 @@ check-declaration-details.declarant-contact-address.label=Declarant contact addr
 #===================================================
 #  SELECT REASON FOR CLAIM PAGE
```

Comment:
this title for this set of messages needs to change as well

Reply:
Do you mean the caption? That's no longer used on the page, unless it's used in the template somewhere?

Reply:
`SELECT REASON FOR CLAIM PAGE` to `SELECT BASIS FOR CLAIM PAGE`

Reply:
Ohhh, you mean the key? Sure...